atom/browser/web_contents_preferences.cc

```diff
@@ -106,6 +106,8 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
   // integration.
   if (IsSandboxed(web_contents))
     command_line->AppendSwitch(switches::kEnableSandbox);
+  if (UsesNativeWindowOpen(web_contents))
```

Comment:
We should just use `if (web_preferences.GetBoolean("nativeWindowOpen", &b) && b)`, calling `UsesNativeWindowOpen()` does unnecessary checks here.